sql_generators/glean_usage/templates/metrics_clients_last_seen.query.sql

```diff
@@ -1,3 +1,18 @@
+{{ header }}
+
+{% raw %}
+{% if is_init() %}
+{% endraw %}
```

Comment:
For consistency (e.g. you specifically changed `baseline_clients_last_seen_v1.query.sql` to switch to this single-line format).
```suggestion
{% raw %}{% if is_init() %}{% endraw %}
```
(ditto for the `{% else %}` and `{% endif %}` parts)